### PR TITLE
SPLICE-729 : Added unit test that make sure distinct is to true in So…

### PR DESCRIPTION
**Pull request by jfilali**
SPLICE-729 : Added unit test that make sure distinct is to true in SortOperation#getDataSet
this is a unit test for SPLICE-724  : ORDER BY with UNION ignored

### CODE REVIEW

File: splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/SortOperationIT.java [hunk 1]
```diff
@@ -140,6 +146,27 @@ protected void starting(Description description) {
...
                     ps.executeBatch();
+
+					ps = spliceClassWatcher.prepareStatement("insert into "+ spliceTableWatcher5+" values (?)");
+					ps.setInt(1,1); ps.execute();
+					ps.setInt(1,1); ps.execute();
+					ps.setInt(1,1); ps.execute();
+					ps.setInt(1,5); ps.execute();
+					ps.setInt(1,2); ps.execute();
+					ps.setInt(1,7); ps.execute();
+					ps.setInt(1,4); ps.execute();
```
Comment (dgomezferro) [inline splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/SortOperationIT.java]: Minor suggestion, feel free to ignore: I'd add a value bigger than some on the other table just to make sure we aren't returning  something like ( ( B order by id) union ( C order by id) )

**Comment by dgomezferro** [comment]
+1

File: splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/SortOperationIT.java (hunk 1 above)
Comment (jfilali) [inline splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/SortOperationIT.java]: Right on, let me add that.

**Comment by jfilali** [comment]
done, rebase and pushed the changes

**Pull request merged**